packages/@react-spectrum/searchwithin/test/SearchWithin.test.js

```diff
@@ -94,34 +94,34 @@ describe('SearchWithin', function () {
   });
 
   it('searchfield and picker are labelled correctly', function () {
-    let {getByRole, getAllByText, getByLabelText} = renderSearchWithin();
+    let {getByRole, getAllByText} = renderSearchWithin();
 
-    let searchfield = getByLabelText('Test', {selector: 'input'});
+    let searchfield = getByRole('searchbox');
     let picker = getByRole('button');
     let group = getByRole('group');
     triggerPress(picker);
 
     let listbox = getByRole('listbox');
     let label = getAllByText('Test')[0];
-    expect(listbox).toHaveAttribute('aria-labelledby', label.id);
-    expect(searchfield).toHaveAttribute('aria-labelledby', label.id);
-    expect(group).toHaveAttribute('aria-labelledby', label.id);
+    expect(listbox).toHaveAttribute('aria-labelledby', `${label.id} ${group.id} ${picker.id}`);
```

Comment:
why doesn't this have the `picker.childNodes[0].id` like below?

Reply:
The `picker.childNodes[0].id` is referencing the span of the picker button that contains the text of the current selected picker option. For the picker menu, I believe it doesn't need to announce the selected picker item since that info is available on the picker button itself (all of this follows the behavior of a standalone picker)

Reply:
Yeah, this is basically because of how Picker does labelling. Related:

https://github.com/adobe/react-spectrum/blob/1e635d5e6540927a001f552194c30d8b3b48d8e8/packages/%40react-spectrum/picker/test/Picker.test.js#L812-L816